include/agentsList.h

```diff
@@ -19,7 +19,7 @@ concept PPStateType = requires (T x) { x.update(); x.gotInfected(); };
 
 class AgentStats {
     public:
-    unsigned infectedTimestamp=0;
+    unsigned infectedTimestamp=UINT32_MAX;
```

Comment:
std::numeric_limits is safer, because unsigned is not predefined 32 bits (mostly yes, but who knows)